docs/migrations/react-scripts-to-craco.md

```
@@ -0,0 +1,323 @@
+# Migration Guide: react-scripts to Craco
+
+**Date:** 2025-11-17  
+**PR:** [#847](https://github.com/Eibon7/roastr-ai/pull/847)  
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)